simulation/node.go

```diff
@@ -614,6 +614,25 @@ func (node *Node) lineReader(reader io.Reader, uartType NodeUartType) {
 	}
 }
 
+// handles an incoming log message from the OT-Node. The policy may be changed in the future,
+// or made configurable.
+func (node *Node) handlerLogMsg(otLevel string, msg string) {
+	switch otLevel {
+	case "D", "I", "N":
+		// only display detailed log msgs if in Node's context or watching that Node.
+		// Otherwise, too much is displayed.
+		if node.S.cmdRunner.GetContextNodeId() == node.Id || node.S.Dispatcher().IsWatching(node.Id) {
+			simplelogger.Infof(msg)
```

Comment:
Should use `Debugf` for "D" ?

Reply:
Yes this makes most sense, but I had the problem that OTNS-internal logging with 'debug' level then also appears which makes it hard to inspect what the OT-node is doing.  
So ideally I want to be able to see OT-node debug level, while not seeing OTNS debug level messages.

Reply:
I think the real issue here might be that we are using a single log level for both node logs and OTNS logs. We may even have another log level for watched nodes.

Reply:
For now this is resolved by using "debug" level for OT node "debug" messages. Having separate levels is indeed the most useful - saving this up for a future PR.